codalab/worker/local_run/docker_image_manager.py

```diff
@@ -160,24 +186,20 @@ def get(self, image_spec):
                         )
                     else:
                         if self._downloading[image_spec]['success']:
-                            image = self._docker.images.get(image_spec)
-                            digest = image.attrs.get('RepoDigests', [image_spec])[0]
-                            with self._lock:
-                                self._image_cache[digest] = ImageCacheEntry(
-                                    id=image.id,
-                                    digest=digest,
-                                    last_used=time.time(),
-                                    virtual_size=image.attrs['VirtualSize'],
-                                    marginal_size=image.attrs['Size'],
-                                )
-                            status = ImageAvailabilityState(
-                                digest=digest, stage=DependencyStage.READY, message='Image ready'
+                            status = image_availability_state(
+                                image_spec,
+                                success_message='Image ready',
+                                failure_message='Image {} was downloaded successfully, but it can not be found locally due to unknown reasons'.format(
+                                    image_spec
+                                ),
                             )
                         else:
-                            status = ImageAvailabilityState(
-                                digest=None,
-                                stage=DependencyStage.FAILED,
-                                message=self._downloading[image_spec]['message'],
+                            status = image_availability_state(
+                                image_spec,
+                                success_message='Image {} can not be downloaded from the DockerHub but it is found locally'.format(
```

Comment:
```suggestion
                                success_message='Image {} could not be downloaded from DockerHub but was found locally'.format(
```

Reply:
@epicfaace Why do we want to change to `could`? We are sure that the image can't be downloaded at this point. Let me know.